js/src/features/project/components/ProjectDescription.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Button } from "antd";
 
-interface Props {
+type Props = {
   description?: string;
   openProjectDescriptionForm: () => void;
```

Comment:
`ProjectDescription` component should not to know about how do you plan to use it.
I think, better will be `onClick` or `onButtonClick`.

Reply:
fix